pycbc/workflow/segment.py

```diff
@@ -1129,6 +1129,7 @@ def get_analyzable_segments(workflow, out_dir, tags=[]):
         curr_file = OutSegFile(ifo, 'SCIENCE_OK', workflow.analysis_time,
                                curr_url, segment_list=sci_segs[ifo],
                                tags=tags)
+        curr_file.PFN(seg_ok_path, 'local')
```

Comment:
Okay, as we're going to fix this properly post 1.2.0 anyway I won't argue on this point.